extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -517,60 +546,75 @@ angular.module('oppia').component('svgFilenameEditor', {
           ctrl.data.savedSvgFileName.length > 0);
       };
 
+      var getStrokeWidth = function() {
+        var size = ctrl.fabricjsOptions.size;
+        // Removes the word "px" from the end of the string.
```

Comment:
It also converts the string to a number, right? (Like '1px' becomes 1) Mention that as well?

Reply:
Also, it seems you are using this as a general function to get number from 'x px', maybe rename the function to something that explains that? Maybe just `getSize()`?

Reply:
Done.